src/writing-your-own-application/generate-project/index.md

```diff
@@ -36,7 +36,7 @@ Both templates are based on [`cargo-generate`][cargo-generate], a tool that allo
 
 ## Using Dev Containers in the Templates
 
-Both template repositories have a prompt for Dev Containers support, see details in [Dev Container][dev-container] section of the template README.
+Both template repositories have a prompt for Dev Containers support, see details in [Dev Containers][dev-container] section of the template README.
```

Comment:
Do you think we need the following refinements for capitalization?

- "Dev Containers" for the [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) VS Code extension
- "dev container" or "dev containers" for refs to actual container instances created in VS Code

Reply:
I noticed that https://containers.dev/ also has different capitalization for Dev [Cc]ontainers.

Eventually, if you think our Vale checks for "Dev Containers" are good for now, then let's just stick with them. We can make updates anytime.